proposals-CG-WG.md

```diff
@@ -29,70 +29,73 @@ The purpose of Stage 0 proposals is to allow anyone to raise and explore the Pro
 
   * What's needed from individuals?
-    * [ ] Identification of [champions](https://github.com/tc39/how-we-work/blob/main/champion.md)
-    * [ ] An understanding of the Problem Space
-    * [ ] Optionally, alternatives under consideration
+    * [ ] Identification of [champions](https://github.com/tc39/how-we-work/blob/main/champion.md).
+    * [ ] An understanding of the Problem Space.
+    * [ ] Optionally, alternatives under consideration.
   * What's asked of the **Community Group**?
-    * [ ] The Community Group consensus that the problem is worth spending the Community Group’s time working on
+    * [ ] The Community Group consensus that the problem is worth spending the Community Group’s time working on.
     * [ ] A home for incubating the proposal. Small features will incubate in issues. If and when the champions are ready to more thoroughly document their proposal, the WG chairs will create a repo for the champions to develop the feature (e.g., [example](https://github.com/fedidcg/LightweightFedCM)). 
 
 # Stage 2: Formalization
 
-The purpose of Stage 2 Proposals is to formally specify the best (and seemingly workable) alternative that was identified in the prior step: handle corner cases, integrate with other parts, reconcile with other proposals, and resolve the concerns identified at the entrance. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.
+The goal of Stage 2 is to refine the preferred solution into a detailed, cohesive proposal that addresses known issues and integrates feedback from stakeholders. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.  This stage focuses on preparing the proposal for Working Group review and creating a complete, formal draft.
 
   * What's needed from champions?
-    * [ ] An [explainer](https://tag.w3.org/explainers/)
-    * [ ] Alternatives and trade-offs considered
-    * [ ] A specific preferred proposal, e.g., code samples, examples, or a spec PRs (if the formalism is needed to understand the proposal)
-    * [ ] Browser and developer implementation experience (e.g., a prototype, dev trials, origin trials, etc) 
-    * [ ] Confidence of developer demand and fitness for purpose (e.g. a developer that needs this proposal)
+    * [ ] An [explainer](https://tag.w3.org/explainers/).
+    * [ ] Documentation of alternatives and trade-offs considered.
+    * [ ] Draft specification text (or detailed examples/code samples if needed for clarity).
+    * [ ] Early implementation experience, including prototypes or trials.
+    * [ ] Evidence of stakeholder demand and use-case alignment.
```

Comment:
Stakeholders seems unnecessarily underspecified as far as "demand" is concerned. Where else would "demand" come from other than web developers?

```suggestion
    * [ ] Evidence of stakeholder (e.g. web developers) demand and use-case alignment.
```

Reply:
Minor tweaks to @samuelgoto's suggestion
```suggestion
    * [ ] Evidence of stakeholder (e.g., web developer) demand and use-case alignment.
```